vm/src/obj/objlist.rs

```diff
@@ -409,6 +409,26 @@ fn list_pop(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     }
 }
 
+fn list_remove(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(
+        vm,
+        args,
+        required = [(list, Some(vm.ctx.list_type())), (needle, None)]
+    );
+
+    let mut elements = get_mut_elements(list);
+    let i = elements
+        .iter()
+        .position(|r| objbool::get_value(&vm._eq(needle.clone(), r.clone()).unwrap()));
+    if i.is_some() {
+        elements.remove(i.unwrap());
+        Ok(vm.get_none())
+    } else {
+        let needle_str = objstr::get_value(&vm.to_str(needle).unwrap());
```

Comment:
This `unwrap` should also be a `?`. This way it'll propagate the exception rather than crashing the interpreter.

Reply:
Done.